pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/ManagedCursorMetricsTest.java

```diff
@@ -197,9 +198,19 @@ public void testManagedCursorMetrics() throws Exception {
         admin.topics().delete(topicName, true);
     }
 
-    @Test
+    private ManagedCursorMXBean getManagedCursorMXBean(String topicName, String subscriptionName)
+            throws ExecutionException, InterruptedException {
+        final PersistentSubscription persistentSubscription =
+                (PersistentSubscription) pulsar.getBrokerService()
+                        .getTopic(topicName, false).get().get().getSubscription(subscriptionName);
+        final ManagedCursorImpl managedCursor = (ManagedCursorImpl) persistentSubscription.getCursor();
+        return managedCursor.getStats();
+    }
+
+    @Test(timeOut = 2 * 1000)
```

Comment:
We can just use the default timeout? Or any reason to set it to 2 seconds for this test? It might become flaky in the CI environment.

Reply:
> Or any reason to set it to 2 seconds for this test?

no

> We can just use the default timeout? 

already change to default timeout.